### PR TITLE
Update/new module kit package

### PR DESCRIPTION
**Pull request by mzpx**

### CODE REVIEW

File: Sources/SystemModule/Adapters/Permission+Adapters.swift [hunk 1]
```diff
@@ -28,7 +28,7 @@ extension System.Permission.Model: CreateAdapter, UpdateAdapter, PatchAdapter {
     }
 }
 
-extension System.Permission.Model.ColumnNames: ColumnNamesInterface {
+extension System.Permission.Model.ColumnNames: ColumnNamesAdapter {
```
Comment (tib) [inline Sources/SystemModule/Adapters/Permission+Adapters.swift]: Shouldn't this be List(Query)SortKeyAdapter? What do you think @viaszkadi ?

File: Sources/SystemModuleKit/Objects/Permission+Objects.swift [hunk 2]
```diff
@@ -21,9 +21,9 @@ extension System.Permission {
 
     public struct List: FeatherModuleKit.List {
```
Comment (tib) [inline Sources/SystemModuleKit/Objects/Permission+Objects.swift]: WDYT: `FeatherModuleKit.List` -> `ListInterface`? cc @viaszkadi

**Review by tib — CHANGES_REQUESTED** [review]

**Review by viaszkadi — APPROVED** [review]

File: Sources/SystemModule/Adapters/Permission+Adapters.swift (hunk 1 above)
Comment (viaszkadi) [reply Sources/SystemModule/Adapters/Permission+Adapters.swift]: thats better

File: Sources/SystemModuleKit/Objects/Permission+Objects.swift (hunk 2 above)
Comment (viaszkadi) [reply Sources/SystemModuleKit/Objects/Permission+Objects.swift]: w can rename `FeatherModuleKit.List` to `FeatherModuleKit.ListInterface` to make these interfaces consistent, that would eliminate naming collision as well

**Review by viaszkadi — APPROVED** [review]

**Review by tib — APPROVED** [review]

**Pull request merged**